spec/controllers/users/totp_setup_controller_spec.rb

```diff
@@ -32,10 +30,6 @@
           subject.current_user.decorate.qrcode(subject.user_session[:new_totp_secret])
         ).not_to be_nil
       end
-
-      it 'presents a QR code to the user' do
```

Comment:
Chatted with @jessieay on Slack -- we agreed this can probably be covered by a view spec that checks for the QR code as an image tag with a data URL